installers/unix/env/install.sh

```diff
@@ -72,6 +72,9 @@ done
 
 check_dep $PYTHON
 
+# We do not support Anaconda using this installer. Anaconda users must install the Engine manually
+python -c "import sys; sys.exit('Anaconda Python isn\'t supported by this package. Please install the official Python from python.org.') if 'conda' in sys.version else ''"
```

Comment:
I'm using ternary operator (with an empty `else`) because `if ... else ...` does not play well with `-c`